.github/workflows/build-test.yml

```diff
@@ -348,23 +348,15 @@ jobs:
         with:
           python-version: ${{matrix.pythonVersion}}
 
-      - name: Cache conda on linux
-        uses: actions/cache@v2
+      - name: Install Conda environment from environment.yml
+        uses: mamba-org/provision-with-micromamba
```

Comment:
I meant uses: `mamba-org/provision-with-micromamba#<commit>`, or whatever the syntax is